db/ens.go

```diff
@@ -165,12 +165,18 @@ func (bigtable *Bigtable) TransformEnsNameRegistered(blk *types.Eth1Block, cache
 
 			nameRegistered, err := filterer.ParseNameRegistered(nameLog)
 			if err != nil {
-				utils.LogError(err, fmt.Sprintf("indexing of register event failed parse register event at index [%v] on block [%v]", foundNameIndex, blk.Number), 0)
+				utils.LogError(err, fmt.Sprintf("indexing of register event failed parse register event at tx [%v] index [%v] on block [%v]", i, foundNameIndex, blk.Number), 0)
 				continue
 			}
+
+			if err = verifyName(nameRegistered.Name); err != nil {
+				logger.Warnf("indexing of register event failed because of invalid name at tx [%v] index [%v] on block [%v]: %v", i, foundNameIndex, blk.Number, err)
```

Comment:
**Suggestion:** I would add the name to this log entry.

Reply:
As these can get really long, I don't want to spam the log file. You can find the event with the other logged informations.